lab/src/mapreduce/common_map.go

```diff
@@ -53,6 +55,29 @@ func doMap(
 	//
 	// Your code here (Part I).
 	//
+
+  	// read input file to buffer
+	file, _ := os.Open(inFile)
```

Comment:
[nit] I think it might be good to check the return status whether there is an error occur when create/read/write file

Reply:
Yea, you're right. I'll be more cautious next time... Thanks for pointing out!